### PR TITLE
add installation instructions for `pex` utility

### PR DESCRIPTION
**Pull request by tdyas**
Add installation instructions for git-machete using the [`pex` utility](https://github.com/pex-tool/pex) which builds single file Python EXecutable environments.

### CODE REVIEW

**Comment by tdyas** [comment]
CI general checks failed because I did not bump the version number. This is a documentation only change though to `README.md` so bumping the version number seems inapplicable.

**Comment by PawelLipski** [comment]
Nice! actually, pls base your PR off `develop`, and then the CI will pass. Approving anyway

**Review by PawelLipski — APPROVED** [review]

**Comment by tdyas** [comment]
> Nice! actually, pls base your PR off `develop`, and then the CI will pass. Approving anyway

Done, rebased off `develop`.

File: README.md [hunk 1]
```diff
@@ -129,6 +129,18 @@ Stable channels may lag behind; see [repology](https://repology.org/project/git-
 
 <br/>
 
+### Using Pex
+
+The [Pex tool](https://github.com/pex-tool/pex) (short for Python EXecutable) allows you to build "pex" files which are executable Python environments in a single file.
```
Comment (PawelLipski) [inline README.md]:
```suggestion
The [Pex tool](https://github.com/pex-tool/pex) (short for Python EXecutable) allows you to build "pex" files which are executable Python environments in a single file.
```

Ouch, trailing space 😅

**Pull request merged**